gym/lib/gym/options.rb

```diff
@@ -280,6 +280,18 @@ def self.plain_options
                                      description: "Sets a custom path for Swift Package Manager dependencies",
                                      type: String,
                                      optional: true),
+        FastlaneCore::ConfigItem.new(key: :skip_resolve_package_dependencies,
+                                     env_name: "GYM_SKIP_RESOLVE_PACKAGE_DEPENDENCIES",
+                                     description: "Skips resolution of Swift Package Manager dependencies",
+                                     type: Boolean,
+                                     optional: true,
+                                     default_value: false),
+        FastlaneCore::ConfigItem.new(key: :disable_automatic_package_resolution,
+                                     env_name: "GYM_DISABLE_AUTOMATIC_PACKAGE_RESOLUTION",
+                                     description: "Prevents packages from automatically being resolved to versions other than those recorded in the `Package.resolved` file",
+                                     type: Boolean,
+                                     optional: true,
+                                     default_value: false),
```

Comment:
I think these 2 option names are very similar and not too distinct. I couldn't tell by their names which does what 😅 
Maybe name them:
`skip_package_dependencies_resolution` - Skips resolution of Swift Package Manager dependencies
`force_package_versioning` or `freeze_package_versioning` or `disable_package_automatic_updates` - "[…] Package.resolved file"

What do you think?

----

Also, optional booleans should be avoided whenever possible imo, and it seems to be a good case here. Since all of those options have default values (false), we can make them non-optional (by simply removing that line) 😊 

----

Could you apply this feedback to all options you created? 🙏

Reply:
`disable_package_automatic_updates` sounds fine :)